.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   # the `deploy` section at the bottom of this file too.
   - 'v10.21.0'
   - 'v12.18.1'
+  - 'v14.12.0'
```

Comment:
Should we put exact versions or major ones only? I think it'd be easier to only list:

```
- 10
- 12
- 14
```

(if that's possible)

Reply:
Renovate will update them automatically: https://github.com/mozilla/addons-linter/commit/8e9da4aa3bcd37b0a1b2810e8e3dcd9862b6663a

Reply:
@willdurand @wagnerand Personally I'm ok with any of the two approaches (either just leave travis to use the last version available or leave renovate bot to manage the exact versions in our travis file), let's just agree on which one we prefer and I'll update the PR accordingly and then merge it.

Reply:
Let's not rely on Renovate for that.